### DISCUSSION TITLE
sortingFn seems like not working well

### DISCUSSION THREAD

**Discussion by WohKX** [KevinVandy/material-react-table#44] [Q&A]
Currently, I am working on customizing the sorting function for certain column. However, it seems like not working well and I am pretty sure that the function could return properly (-1, 0, 1). The code snippet is as below: 

`
const c_DateSortingFn = (rowA, rowB, col) => {

  var dateA = rowA.original[col];
  var dateB = rowB.original[col];
  var _dateA = moment(dateA, "DD/MM/YYYY hh:mm:ss A", true);
  var _dateB = moment(dateB, "DD/MM/YYYY hh:mm:ss A", true);
 return _dateA.isAfter(_dateB) ? -1 : _dateB.isAfter(_dateA) ? 1 : 0;
}

const columns = [
    ...
    {
      accessorKey: "UPDATE_DATETIME",
      header: "Last Update",
      size: 200,
      Cell:({cell}) => (cell.getValue() === null) ? (<><label> - </label></>) : cell.getValue(),
      sortingFn : (rowA, rowB, columnId) => {c_DateSortingFn(rowA, rowB, columnId)}
    },
    ...
  ];`

Did I missed out anything? Thanks for your time.

**Comment by KevinVandy** [comment]
Sorry, I'd need more specifics than "not working well" in order to help. If you are providing your own sortingFn, it is up to that code to perform sorting correctly. Make a code sandbox reproducing the issue and I could help.

**Reply by WohKX** [reply]
Yes, I am providing my sortingFn and here is the link for code sandbox: 
https://codesandbox.io/s/material-react-table-sample-custom-sortingfn-forked-spk8d3

**Reply by KevinVandy** [reply]
check out the [advanced example](https://www.material-react-table.com/docs/examples/advanced) for advice on how to sort dates

**Reply by WohKX** [reply]
Currently my requirement was achieved according to the example. Thanks!
